kmod/src/kernelcompat.h

```diff
@@ -172,6 +172,18 @@ do {					\
 #define kc_bio_get_errno(bio)			({ (int)((void)(bio), _error_arg); })
 #endif
 
+/*
+ * MS_* flags from <linux/mount.h> should not be used in the kernel
+ * anymore from 4.x onwards. Instead, we need to use the SB_* versions
+ */
+#ifdef SB_POSIXACL
+#define KC_MS_POSIXACL SB_POSIXACL
```

Comment:
Can we just use the SB_ definitions directly in our code and fall back to the MS_ ones in older code?